pkgs/servers/web-apps/weblate/default.nix

```diff
@@ -0,0 +1,66 @@
+{ lib, fetchFromGitHub, poetry2nix, gettext, isocodes, xorg }:
+
+(poetry2nix.mkPoetryApplication {
+  src = fetchFromGitHub {
+    owner = "WeblateOrg";
+    repo = "weblate";
+    rev = "weblate-4.14";
+    sha256 = "sha256-PsFYL/yKu90QQ0/XdbVFHY1uTU/6H+YfitqFb6Ic4E8=";
+  };
+  pyproject = ./pyproject.toml;
+  poetrylock = ./poetry.lock;
```

Comment:
Can we upstream these?

Reply:
I never tried it, but I'd like to, yes. Could imagine upstream not to be very willing to change their build system / maintain a second one though.

Reply:
Maybe open an issue explaining the needs for lock files first?